tools/release-docs-scripts/pre-conditions.js

```diff
@@ -0,0 +1,21 @@
+import 'colors';
+import { exec } from 'child-process-promise';
+import releasePreConditions from '../release-scripts/pre-conditions';
+
+const docBranchName = 'docs';
+
+function ensureBranchDocs() {
+  return exec('git symbolic-ref --short HEAD')
+    .then(result => {
+      let currentBranchName = result.stdout.trim();
+      if (currentBranchName !== docBranchName) {
+        throw new Error(`Must be on ${docBranchName} branch to do docs-only releases. Currently: ${currentBranchName}`);
+      }
+      console.info(`On ${docBranchName} branch`.cyan);
+    });
+}
+
+export default function docsReleasePreConditions() {
+  return releasePreConditions().then(ensureBranchDocs);
+}
+
```

Comment:
I don't think we need to enforce the name of the branch. This part can just be a documentation point.